packages/calcite-components/src/components/checkbox/checkbox.scss

```diff
@@ -4,16 +4,31 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-checkbox-size: Specifies the component's height and width.
+ * @prop --calcite-checkbox-box-shadow: Specifies the component's color.
```

Comment:
Upldate this to `--calcite-checkbox-box-shadow-color`

Reply:
I thought we were using `shadow`, not `box-shadow` here? https://github.com/Esri/calcite-design-system/wiki/component-tokens#3-review-component-for-tokenizable-styles

Reply:
I renamed this one to `--calcite-checkbox-shadow-color`, as suggested in the doc above.

Reply:
I realize it's basically being used as a border - but that begs the question.. Shouldn't this be a separate `--aclcite-checkbox-border-color` (obfuscating the implementation detail of it using box-shadow from the user) - and then a separate `calcite-checkbox-shadow` css prop that defaults to `shadow-none` to allow for theming?

Reply:
@macandcheese Right again. I thought these new props were used for a drop shadow, but I overlooked how they were being applied.

@aPreciado88 The shadow props will need to be updated. The `themed` helper handles this and another case [here](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/tests/commonTests/themed.ts#L405-L408). You can use this as a reference to confirm if we’re dealing with a border prop or not. LMK if you have any questions.

Sorry for the hassle, y'all!

Reply:
I updated the tokens to `--calcite-checkbox-border-color` and `--calcite-checkbox-border-color-hover`.

Reply:
Yeah I'm sorry. I agree with @macandcheese and @jcfranco